### ISSUE TITLE
Missing keypoint coordinate when exporting dataset into coco keypoint 1.0

### ISSUE THREAD

**Issue by chenmz-lab** [cvat-ai/cvat#8484]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://docs.cvat.ai/docs/)

### Is your feature request related to a problem? Please describe.

When exporting the dataset(which I had annotated) into coco keypoint 1.0 format,I found that the json is missing the coordinate data of the keypoints,inversely,there were just names of the keypoints,did I misunderstood the function of cvat?
![keypoint missing](https://github.com/user-attachments/assets/58a7eb10-f2a3-4f84-9d5e-3a2ca3145f46)

**Comment by bsekachev** [comment]
Hello,

What is expected COCO keypoints format? May you refer to official documentation that would say that exported file from CVAT is incorrect?

**Comment by chenmz-lab** [comment]
> Hello,
> 
> Could you kindly clarify the expected format for COCO keypoints? Additionally, would you be able to point me to the official documentation that indicates the exported file from CVAT may not be in the correct format?
> 
> Thank you!
Yes!Here is the official documentation dataset from COCO keypoints annotations:We can see that in this format,the keypoints data are coordinate information,the first two number represent the location of the point and the third number represents if the keypoint is occluded.
![image](https://github.com/user-attachments/assets/b4e8612f-cc57-4442-928c-10d7a0cd37cd)

**Comment by zhiltsov-max** [comment]
Please try to scroll the file down. The section with annotations goes after "categories", which are shown on the screenshot in the issue description.
The format specification is available here: https://cocodataset.org/#format-data

**Comment by chenmz-lab** [comment]
> Please try to scroll the file down. The section with annotations goes after "categories", which are shown on the screenshot in the issue description. CVAT can export skeleton annotations in this format. The format specification is available here: https://cocodataset.org/#format-data

Thanks for your reply!Unfortunately,later I found that the problem was when I annotated the picture,I chose the "track" rather than "shape",when I switched to "shape" then the exported json file had the "keypoints number"
